angular-webflux/package.json

```diff
@@ -48,16 +48,16 @@
     "e2e:devserver": "concurrently -k -s first \"npm run backend:start\" \"npm start\" \"wait-on http-get://localhost:9000 && npm run e2e:headless -- -c baseUrl=http://localhost:9000\"",
     "pree2e:headless": "npm run ci:server:await",
     "e2e:headless": "npm run e2e:cypress --",
-    "java:docker": "./mvnw -ntp verify -DskipTests -Pprod jib:dockerBuild",
+    "java:docker": "./mvnw -ntp verify -DskipTests -Pnative,prod jib:dockerBuild",
     "java:docker:arm64": "npm run java:docker -- -Djib-maven-plugin.architecture=arm64",
-    "java:docker:dev": "npm run java:docker -- -Pdev,webapp",
-    "java:docker:prod": "npm run java:docker -- -Pprod",
+    "java:docker:dev": "npm run java:docker -- -Pnative,dev,webapp",
+    "java:docker:prod": "npm run java:docker -- -Pnative,prod",
     "java:jar": "./mvnw -ntp verify -DskipTests --batch-mode",
-    "java:jar:dev": "npm run java:jar -- -Pdev,webapp",
-    "java:jar:prod": "npm run java:jar -- -Pprod",
+    "java:jar:dev": "npm run java:jar -- -Pnative,dev,webapp",
+    "java:jar:prod": "npm run java:jar -- -Pnative,prod",
     "java:war": "./mvnw -ntp verify -DskipTests --batch-mode -Pwar",
-    "java:war:dev": "npm run java:war -- -Pdev,webapp",
-    "java:war:prod": "npm run java:war -- -Pprod",
+    "java:war:dev": "npm run java:war -- -Pnative,dev,webapp",
+    "java:war:prod": "npm run java:war -- -Pnative,prod",
```

Comment:
@mshima A native app currently doesn't work with the `dev` profile, so I'm not sure we should add `native` to `dev` profiles.

Reply:
Ok, let’s drop.